docs/tips_tricks_hacks.md

```diff
@@ -0,0 +1,34 @@
+# Tips Tricks and Hacks
```

Comment:
@x85446 As this is not related to this PR, so we should remove it and bring it in through another PR.

Reply:
removed with subsequent commit.